src/controllers/albumController.js

```diff
@@ -185,7 +188,31 @@ const getFilteredAlbums = async (req, res) => {
     .skip(parseInt(offset) || 0) // Skip a specified number of albums (pagination implementation)
     .limit(parseInt(limit) || 10); // Limit the number of returned albums (pagination implementation)
 
-  res.status(StatusCodes.OK).json({ albums, count: albums.length }); // Return the found albums and the count of albums
+  const totalPages = Math.ceil(totalCount / (parseInt(limit) || 10));
+  const currentPage = Math.ceil(
+    (parseInt(offset) + 1) / (parseInt(limit) || 10)
```

Comment:
```suggestion
    ((parseInt(offset) || 0) + 1) / (parseInt(limit) || 10)
```

^ in case `offset` was not given in the query params and is undefined/null